stacks/summer-protocol.ts

```
@@ -0,0 +1,44 @@
+import { SummerStackContext } from './summer-stack-context'
+import { Cron, Function, FunctionProps } from 'sst/constructs'
```

Biome (1.9.4)

[error] Do not shadow the global "Function" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)